docs/planification.rst

```diff
@@ -2,9 +2,10 @@
 Package Structure
 =================
 
-Here is `xhydro`'s structure as of v0.3.3 (2024-02-22), with the planned or discussed additions for the future.
+Here is `xhydro`'s structure as of 2024-06-07, with the planned or discussed additions for the future.
```

Comment:
I would use the dev version string with the date if the changes are between stable releases.